docs/design/autoware-interfaces/components/perception.md

```diff
@@ -0,0 +1,35 @@
+# Perception
+
+![Node diagram](images/Perception-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### PointCloud
+
+PointCloud data published by Lidar.
+
+- [sensor_msgs/PointCloud2](http://docs.ros.org/en/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Image
+
+Image frame captured by camera.
+
+- [sensor_msgs/Image](http://docs.ros.org/en/api/sensor_msgs/html/msg/Image.html)
+
+### Vehicle kinematic state
+
+current position of ego, used in traffic signals recognization. See Iutputs of Planning.
+
+### Lanelet2 Map
+
+map of the environment. See Iutputs of Planning.
+
+## Output
+
+### 3D Object Predictions
+
+3D Object detected, tracked and predicted by sensor fusing. See Iutputs of Planning.
+
+### Traffic Light Response
+
+traffic light signals recognized by object detection model. See Iutputs of Planning.
```

Comment:
Besides these, the following two are also used as Perception outputs.
- Occupancy Grid Map : Mainly to handle blind spots in Planning
- Obstacle Segmentation Point Cloud : Includes not only dynamic objects but also stationary objects. This is used in Planning if if there are any obstacles to be hit.